Applications/Spire/Include/Spire/Charting/ChartView.hpp

```diff
@@ -95,9 +141,28 @@ namespace Spire {
       bool m_is_auto_scaled;
       QtPromise<std::vector<Spire::Candlestick>> m_candlestick_promise;
       std::vector<Candlestick> m_candlesticks;
+      QCursor m_hand_cursor;
```

Comment:
Look into moving some of these members out of the class. Chances are every view uses the same QCursor, so use a function:

```
namespace {
  auto get_hand_cursor() {
    static auto cursor = ...;
    return cursor;
  }
}
```

Reply:
Where would these go if they were in an anonymous namespace? Shouldn't it be the Ui header and source files?

Reply:
ChartView.cpp